extensions/mssql/package.json

```diff
@@ -383,6 +383,11 @@
           "type": "boolean",
           "description": "%mssql.ignorePlatformWarning%",
           "default": false
+        },
+        "mssql.parallelMessageProcessing": {
+          "type": "boolean",
+          "description": "%mssql.parallelMessageProcessing%",
+          "default": false
```

Comment:
I think it'd be a good idea to : 

1. Force this to true for dev builds (easy enough to revert if we start hitting issues)
2. Wait a week or so and if there's no major issues default to true on insiders

It'll be important to get people testing this as much as possible - and if we just leave it as default disabled then I imagine we're not really going to get much actual testing done.

Reply:
that sounds good to me, I was planning to ask folks to turn it on, i guess have it on for dev build by default is a better idea :)

Reply:
having per quality level default value is actually not as straightforward as I was expecting. in ADS core, the configuration default is in code file, we can check the quality there, but this one is in package.json. we have to rely on the mixing step to change the value. 

are you aware of easier approaches? if no, I'd probably go with the original plan.

Reply:
Yeah, I wish there was a way to make it better. The only solution I can come up with is : 

1. Have a config flag `setDefaultParallelMessageProcessingValue` that you check when extension starts up
2. If it doesn't exist or is true and quality is insiders then update the config value and set `setDefaultParallelMessageProcessingValue` to false

That way it only does it once - so if the user then changes the value themselves it keeps that new value forever (unless they wipe our their settings)

Not pretty but it'll work.